modules/admin_manual/pages/configuration/files/manual_file_locking.adoc

```diff
@@ -51,14 +51,17 @@ Using the occ command::
 
 == Configuration
 
-To prevent files being locked infinitely, there is a mechanism that automatically expires locks after a certain time. The expiration time of locks can either be configured via the web interface or using occ commands:
+To prevent files being locked infinitely, there is a mechanism that automatically expires locks after a certain time. The expiration time of locks can either be configured via the web interface or using occ commands. In addition, administrators can define groups which members can break locks.
```

Comment:
```suggestion
To prevent files being locked infinitely, there is a mechanism that automatically expires locks after a certain time. The expiration time of locks can either be configured via the web interface or using occ commands. In addition, administrators can define groups whose members can break locks.
```

Reply:
Actually I am not sure about this one! "groups which members" sounds strange to me, but a sentence like "these are the groups which have a lot of members" sounds fine. "which" is about "groups", and the group is not a person, so "which" should be correct. I need an English professional!

Reply:
https://ell.stackexchange.com/questions/164967/which-vs-whose has a good explanation. "whose" is correct in this context. "who" is a word asking about a living thing, but "whose" can be used referring to both living and non-living things.